docs/api/supervisor/endpoints.md

```diff
@@ -2057,8 +2057,15 @@ Returns information about available updates
 
 </ApiEndpoint>
 
+<ApiEndpoint path="/reload_updates" method="post">
+This reloads information about main components (OS, Supervisor, Core, and
+Plug-ins).
```

Comment:
```suggestion
add-ons).
```
should this be "add-ons" instead of Plug-ins?

Reply:
Nope, this reload the plugins :)